pkgs/os-specific/bsd/freebsd/pkgs/ldd.nix

```diff
@@ -0,0 +1,21 @@
+{
+  lib,
+  stdenv,
+  mkDerivation,
+  libelf,
+}:
+mkDerivation ({
+  path = "usr.bin/ldd";
+  extraPaths = [
+    "libexec/rtld-elf"
+    "contrib/elftoolchain/libelf"
+  ];
+
+  buildInputs = [ libelf ];
+
+  env = {
+    NIX_CFLAGS_COMPILE = "-D_RTLD_PATH=${lib.getLib stdenv.cc.libc}/libexec/ld-elf.so.1";
+  };
+
+  meta.platforms = lib.platforms.freebsd;
+})
```

Comment:
I think the parens are unnedeed?

Reply:
Parens were added months ago as part of a mass rebuild avoidance hack that I just found and removed prior to this PR, but not formatting-wise!